secure-mobile-push/smp-android-client/securemobilepush/src/main/java/de/adorsys/android/securemobilepush/sms/SmsReceiver.java

```diff
@@ -0,0 +1,71 @@
+package de.adorsys.android.securemobilepush.sms;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.content.LocalBroadcastManager;
+import android.telephony.SmsMessage;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import de.adorsys.android.securemobilepush.BuildConfig;
+
+public class SmsReceiver extends BroadcastReceiver {
+    public static final String INTENT_FILTER_SMS = "intentFilterSms";
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/commit/956ba7e93c71c09d0ecf588c93883c4cacb8740a